recipe/meta.yaml

```diff
@@ -175,6 +265,179 @@ outputs:
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      ignore_run_exports:
+        - cudatoolkit    # [cuda_compiler_version not in (undefined, "None")]
+    requirements:
+      build:
+        # c compiler is specified here to get run constraint pins correct, presumably..
+        - {{ compiler('c') }}
+        - {{ compiler('cgo') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+        - clangdev {{ llvm_version }}
+        # clang++ is used for generating the bytecodes of extension functions
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - make
+        - ninja
+        - maven
+      host:
+        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
+        - bisonpp
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - double-conversion
+        - flex
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libgdal
+        - libpng
+        - librdkafka
+        - tbb
+        - tbb-devel
+        - thrift-cpp {{ thrift_version }}
+      run:
+        - arrow-cpp-proc {{ arrow_version }} {{ build_ext }}
+        - boost-cpp
+        - bzip2
+        # omnscidb Load-time UDF support calls clang++
+        - gxx_{{ target_platform }}
+        - libclang-cpp {{ llvm_version }}
+        - ncurses
+        - openjdk 8.*
+        - xz
+        - zlib
+        - {{ pin_subpackage('omniscidb-common',  max_pin='x.x.x') }}
+      run_constrained:
+        - arrow-cpp-proc {{ arrow_version }} {{ build_ext }}
+        - cudatoolkit >=11.0  # [cuda_compiler_version not in (undefined, "None")]
+
+    test:
+      commands:
+        # Test installation
+        - test -f ${PREFIX}/lib/libDBEngine.so
+        # Verify the activation scripts are in-place.
+        {% for state in ["activate", "deactivate"] %}
+        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"
+        {% endfor %}
+        - |
+          source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
+          test -n "${OMNISCI_ROOT_PATH+x}"
+
+    about:
+      home: https://www.omnisci.com/
+      license: Apache-2.0
+      license_family: APACHE
+      license_file: LICENSE.md
+      summary: The OmniSci database
+
+      description: |
+        OmniSciDB is an in-memory, column store, SQL relational database
+        that was designed from the ground up to run on GPUs.
+      doc_url: https://www.omnisci.com/docs/latest/
+      dev_url: https://github.com/omnisci/omniscidb
+
+  - name: pyomniscidbe
+    version: {{ version }}
+    script: build-omniscidb.sh
+    build:
+      string: {{ py_build_string_ext }}
+      # tbb4py is restricted to py<=38
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64 or py>=39]
```

Comment:
@pearu any reason to avoid python 3.9?

Reply:
yes, tbb4py conda package is not available for python 3.9 yet, see https://github.com/conda-forge/tbb-feedstock

Reply:
Ah, right, thanks!